.travis.yml

```diff
@@ -12,3 +24,17 @@ before_script:
 script:
   - yarn test:coverage
   - yarn codecov
+  - export BRANCH=$(if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then echo "$TRAVIS_BRANCH"; else echo "$TRAVIS_PULL_REQUEST_BRANCH"; fi)
+  - if [ "$BRANCH" == "master" ] && [ "$TRAVIS_PULL_REQUEST" == "false" ]; then ./release.sh compare_and_release; fi
+
+jobs:
+  include:
+    - stage: test
```

Comment:
stage: test?

Reply:
This is default stage name. I was trying to test some conditioning but it turns out it would work same anyway with any stage name. reverting to deploy